configuration/configuration.go

```diff
@@ -10,19 +10,29 @@ import (
 )
 
 const (
-	defaultListenAddress               string = "0.0.0.0"
-	defaultListPort                    int    = 2803
-	defaultRulesFile                   string = "/etc/falco-talon/rules.yaml"
-	defaultWatchRules                  bool   = true
-	defaultPrintAllEvents              bool   = false
-	defaultDeduplicationLeaderElection bool   = true
-	defaultDeduplicationTimeWindow     int    = 5
+	defaultListenAddress                string = "0.0.0.0"
+	defaultListPort                     int    = 2803
+	defaultRulesFile                    string = "/etc/falco-talon/rules.yaml"
+	defaultWatchRules                   bool   = true
+	defaultPrintAllEvents               bool   = false
+	defaultDeduplicationLeaderElection  bool   = true
+	defaultDeduplicationTimeWindow      int    = 5
+	defaultOtelCollectorEndpoint        string = "localhost"
+	defaultOtelCollectorUseInsecureGrpc bool   = false
+	defaultOtelCollectorPort            int    = 4317
 )
 
+type Otel struct {
+	CollectorEndpoint        string `mapstructure:"collector_endpoint"`
+	CollectorPort            string `mapstructure:"collector_port"`
+	CollectorUseInsecureGrpc bool   `mapstructure:"collector_use_insecure_grpc"`
+}
```

Comment:
The Endpoint should allow `ip:port` or `http(s):domain(:port)/path`, to work with the local collectors and SaaS (eg, Tempo of Grafana Cloud). It also means we should drop `collector_port` setting, cause it becomes useless.
For SaaS and other, we should allow to specify user/password, for now, I can't test with my Grafana Cloud account.

Reply:
I didn't know that it was possible to use a external collector instance, does that only work with Grafana cloud? I'll try create a free account and change this.